test/unit_Interval.jl

```diff
@@ -93,6 +93,12 @@ for N in [Float64, Float32, Rational{Int}]
     E = intersection(A, Interval(N(0), N(1)))
     @test isempty(E)
 
+    # disjointness check
+    @test !isdisjoint(A, B)
+    @test isdisjoint(A, Interval(N(-1), N(1)), false)
+    s, w = isdisjoint(A, B, true)
+    @test s == false && w ∈ A && w ∈ C
```

Comment:
```suggestion
    @test s == false && w ∈ A && w ∈ B
```

Reply:
Oh i had forgotten to fix that typo, thanks 🕵  :